_posts/2022-05-18-#24148.md

```diff
@@ -54,6 +56,8 @@ This is a 2-part Review Club. Since we haven't covered Miniscript before, we fir
 
```

Comment:
(a few lines up)

> How do we keep track of a `Node`'s type and type properties?

Is this a type-o, maybe missing a link for disambiguation, or am I missing something? 😅

Reply:
I've added in hyperlinks to avoid ambiguity, does this clear it up?